src/sonic-yang-models/yang-models/sonic-flex_counter.yang

```diff
@@ -50,9 +50,6 @@ module sonic-flex_counter {
                 leaf FLEX_COUNTER_STATUS {
                     type flex_status;
                 }
-                leaf FLEX_COUNTER_DELAY_STATUS {
```

Comment:
Can we put off the YANG model change? I am wondering if there is gap between the config change and yang model change, which will cause GCU failure in PROD

Reply:
@wen587 Who is setting FLEX_COUNTER_DELAY_STATUS field? I removed it from the code base, so removing it from the yang model.

Reply:
I mean during config migration. If a device was on an image which has FLEX_DELAY field in old config, if we boot up with new image it will carry over the old config which has FLEX_DELAY field. 
However, the YANG model will fail to recognize this field because of the removal.

Reply:
@wen587 Please check utilities PR https://github.com/sonic-net/sonic-utilities/pull/3577.
It handles this flow in db_migrator.py. It should remove this field during DB migration.

Reply:
@wen587 Discussed offline, there's a bypass of db migrator, thus reverted yang model change.

Reply:
@wen587 Yes